lib/npm-check-updates.js

```diff
@@ -420,12 +418,14 @@ process.on('unhandledRejection', err => {
 async function run(options = {}) {
     // if not executed on the command-line (i.e. executed as a node module), set some defaults
     if (!options.cli) {
-        options = _.defaults({}, options, {
+        const defaultOptions = {
+            ...options,
```

Comment:
The `defaultOptions` should not include `options` since it is merged in the next statement.

Reply:
Oops, typo, will fix ASAP.